README.md

```diff
@@ -34,11 +34,27 @@ make
 make install # Might require super user permissions
 ```
 
-Then restart PostgreSQL and create the extension in the `psql` CLI:
+Make sure that you have `pg_config` in your path.
+
+Edit `postgresql.conf` to include the `pg_prometheus` extension:
+
+```
+shared_preload_libraries = 'pg_prometheus'
+```
+
+Start PostgreSQL and install the extension as a superuser using the `psql` CLI:
 ```SQL
 CREATE EXTENSION pg_prometheus;
 ```
 
+Optionally grant permissions to the database user (`prometheus`) that will own the Prometheus data:
```

Comment:
Maybe we should also include the command to create user/role `prometheus` (optionally), because I guess the command bellow will fail if there is no role `prometheus`.